apps/extension/src/senders/validate.ts

```diff
@@ -13,6 +13,10 @@ type ValidSender = chrome.runtime.MessageSender & {
   url: `${ValidProtocol}//${string}/${string}`;
```

Comment:
the origin/url type are incorrect now that validprotocol may conditionally be http at any moment. probably just set these to 'string'